assets/javascripts/kitten/components/cards/triptych.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component } from 'react'
+import Radium, { StyleRoot } from 'radium'
+import { Grid, GridCol as GridColBase } from 'kitten/components/grid/grid'
+import { Marger as MargerBase } from 'kitten/components/layout/marger'
+import { Title } from 'kitten/components/typography/title'
+import { VerticalStroke } from 'kitten/components/layout/vertical-stroke'
+import { ScreenConfig } from 'kitten/constants/screen-config'
+import { mediaQueries } from 'kitten/hoc/media-queries'
+import { debounce } from 'kitten/helpers/utils/debounce'
+
+const Marger = Radium(MargerBase)
+const GridCol = Radium(GridColBase)
+const width = 349
+const height = 465
+
+class FakeCard extends Component {
+  render() {
+    const {
+      index,
+      viewportIsTabletOrLess,
+      viewportIsMobile,
+      ...others
+    } = this.props
+
+    return (
+      <div {...others}>
+        <img
+          src={`https://picsum.photos/${width}/${height}`}
+          style={{
+            width: '100%',
+            height: 0,
+            paddingBottom: `${height / width * 100}%`,
+            background: `transparent url(https://picsum.photos/${width}/${height}) no-repeat`,
+            backgroundSize: 'cover',
+          }}
+          alt={`picture ${index}`}
+        />
+        <h3>{index} / title goes here</h3>
+        <p>some lorem ipsum</p>
+      </div>
+    )
+  }
+}
+
+class TriptychBase extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { addEvenMargin: 0 }
+  }
+
+  manageCardPosition = () => {
+    this.setState((state, props) => {
```

Comment:
Voilà pourquoi j'ai utilisé cette solution :
https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous